java/src/jmri/jmrit/beantable/LogixNGModuleTableAction.java

```diff
@@ -113,12 +85,12 @@ public LogixNGModuleTableAction() {
 
     @Override
     protected void enableAll(boolean enable) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+        JOptionPane.showMessageDialog(null, "Module enable is not implemented yet.", "Error", JOptionPane.ERROR_MESSAGE);
```

Comment:
A message dialog is much better than throwing an exception, since the user can click on the `enable` checkbox. But it would be even better to either disable or hide the `enable` checkbox for Modules and Tables. I did this since I wasn't sure about the details about the bean tables and I didn't had time to dig into it.

Modules cannot be enabled/disabled. A module is like a Java function. If you have a module, you can call it. So enable/disable has no meaning for a Module.

The same applies for LogixNG Tables. They too has no concept of enable/disable.

But this doesn't need to be resolved now if you prefer to focus on other things.

Reply:
- The Options menu is not created for the Modules and Tables.
- Manual clicks on the Enable checkboxes for Modules and Tables are ignored.
- I did not take the time to figure out how to hide the column.